packages/xo-web/src/common/xo/copy-vms-modal/index.js

```diff
@@ -173,6 +174,11 @@ class CopyVmsModalBody extends BaseComponent {
 
 CopyVmsModalBody.PropTypes = {
   vms: PropTypes.arrayOf(PropTypes.string).isRequired,
+  type: PropTypes.oneOf(['VM', 'VM-snapshot', 'VM-template']),
```

Comment:
This PR is not dedicated to copy/clone `VM-snapshot`.

Reply:
`CopyVmsModalBody` is used to copy VM snapshot.